docs/advanced.rst

```diff
@@ -34,6 +34,8 @@ Optionally, you can override the following methods:
 
 - ``get_autocomplete_queryset(self, queryset, q)``: a method that allows the user to provide custom filtering for the autocomplete view. By default, this filters to Accounts whose email contains the case-insensitive search string in ``q``.
 - ``get_autocomplete_label(self, account)``: a method that allows the user to set the label for an account shown in forms using the autocomplete widget.
+- ``after_account_link_verify(self, user)``: a method to perform any custom actions after an account is successfully linked.
```

Comment:
Should say something about exception handling as well, eg

"If an exception is raised by this method, account linking will still continue and the `account_verify_notification_email` email will be notified (if set)."